### ISSUE TITLE
How to get the 2D mask and its ground truth on the BEV plane？

### ISSUE THREAD

**Issue by Ignite616** [Robertwyq/Frustum#2]
May I ask if your 2D mask is extracted with the pre-trained mask rcnn model?
In your paper, you use the 3D box ground truth to supervise the projection of the 2D mask onto the bev plane. However, the projection of the 2D mask on the bev plane is irregular. Is it necessary to find the minimum circumscribed rectangle first, and then calculate the loss with the true value?
Why use focal loss instead of IOU loss?

**Comment by Ignite616** [comment]
May I ask your model, using Resnet101 backbone, image resolution 1600×640. How long does it take to train for 20 epochs on 8 NVIDIA A100 GPUs with batch size 1 per GPU?

**Comment by Robertwyq** [comment]
About 3 days
